packages/website/components/table/table.js

```diff
@@ -2,16 +2,19 @@ import React, { useCallback } from 'react';
 
 import Loading from 'components/loading/loading';
 import Dropdown from 'ZeroComponents/dropdown/dropdown';
-import Pagination from 'components/table/pagination';
+// import Pagination from 'components/table/pagination';
 import SelectCell from 'components/table/selectCell';
+import { ServerPagination } from 'ZeroComponents/pagination/pagination';
+import AppData from 'content/pages/app/account.json';
+
+const fileRowLabels = AppData.page_content.file_manager.table.file_row_labels;
 
 /**
  * @typedef {Object} ColumnDefinition
  * @property {string | import('react').ReactComponentElement } headerContent
  * @property {string} id
  * @property {import('react').FC} [cellRenderer]
```

Comment:
```suggestion
 * @property {import('react').FC<any>} [cellRenderer]
```

To not have checker complain about prop types.

Reply:
Aha! I saw playing with so many types to get this to work... I totally missed this. Thanks!